data_steward/cdr_cleaner/cleaning_rules/deid/fitbit_device_id.py

```diff
@@ -0,0 +1,138 @@
+"""
+deidentify 'device_id' in RT and CT
+
+Original Issue: DC-3262
+
+The new fitbit device table contains a device_id field that has a privacy requirement associated
+with it. Every person_id/device_id pair should be given a unique id that will be stable across CDR
+versions.
+
+A lookup table will have been created in pipeline_tables. DC-3262: Create the `
+pipeline_tables.wearables_device_id_masking` table
+
+A cleaning rule will have been created to keep the lookup table updated at each run of the CDR.
+DC-3229: Create a CR to update 'device_id_mapping' in pipeline_tables
+
+"""
+import os
+import logging
+
+from common import PIPELINE_TABLES, JINJA_ENV
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+
+LOGGER = logging.getLogger(__name__)
+
+DEVICE_ID = 'device_id'
+
+DEID_FITBIT_DEVICE_ID = JINJA_ENV.from_string("""
+UPDATE `{{project_id}}.{{dataset_id}}.device` AS d
+SET d.{{device_id}} = wdim.research_device_id
+FROM `{{project_id}}.{{pipeline_tables}}.wearables_device_id_masking` AS wdim
+WHERE d.person_id = wdim.person_id
+AND d.{{device_id}} = wdim.{{device_id}}
+AND wearable_type = 'fitbit'
+""")
+
+ISSUE_NUMBERS = ['DC-3254']
+
+
+class DeidFitbitDeviceId(BaseCleaningRule):
+    """
+    Every person_id/device_id pair should be given a unique ID
+    """
+
+    def __init__(self, project_id, dataset_id, sandbox_dataset_id):
+        """
+        Initialize the class with proper info.
+
+        Set the issue numbers, description and affected datasets.  As other
+        tickets may affect this SQL, append them to the list of Jira Issues.
+        DO NOT REMOVE ORIGINAL JIRA ISSUE NUMBERS!
+        """
+
+        desc = (
+            """Every person_id/device_id pair should be given a unique id """
```

Comment:
The actual description will be 

"Updates reasearch_device_id with the device_id in the device table."